cli/logs.go

```diff
@@ -0,0 +1,89 @@
+package cli
+
+import (
+	"context"
+	"errors"
+	"io"
+	"log"
+	"strings"
+
+	"github.com/odpf/salt/term" //nolint
+	entropyv1beta1 "go.buf.build/odpf/gwv/odpf/proton/odpf/entropy/v1beta1"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/odpf/salt/printer"
+	"github.com/spf13/cobra"
+)
+
+func cmdLogs(ctx context.Context) *cobra.Command {
+	var urn string
+	var filter []string
+	filters := make(map[string]string)
+	cmd := &cobra.Command{
+		Use:     "logs",
+		Aliases: []string{"logs"},
+		Short:   "Gets logs",
+		Example: heredoc.Doc(`
+			$ entropy logs
+		`),
+		Annotations: map[string]string{
+			"action:core": "true",
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			spinner := printer.Spin("")
+			defer spinner.Stop()
+			cs := term.NewColorScheme()
+
+			client, cancel, err := createClient(cmd)
+			if err != nil {
+				return err
+			}
+			defer cancel()
+
+			var reqBody entropyv1beta1.GetLogRequest
+			for _, f := range filter {
+				keyValue := strings.Split(f, ":")
+				filters[keyValue[0]] = keyValue[1]
+			}
+
+			reqBody.Filter = filters
+			reqBody.Urn = urn
+
+			err = reqBody.ValidateAll()
+			if err != nil {
+				return err
+			}
+
+			stream, err := client.GetLog(ctx, &reqBody)
+			if err != nil {
+				return err
+			}
+			spinner.Stop()
+
+			done := make(chan bool)
+
+			go func() {
```

Comment:
Do we need this go routine if we are blocking just after?

Reply:
same question here: https://github.com/odpf/entropy/pull/53/files#r884498085